config.json

```diff
@@ -820,6 +820,25 @@
           "math",
           "algorithms"
         ]
+      },
+      {
+        "uuid": "f26bf680-b8fb-40dd-a82c-884ae391b8b6",
+        "practices": [],
+        "prerequisites": [],
+        "slug": "saddle-points",
+        "name": "Saddle Points",
+        "difficulty": 4,
```

Comment:
Let me know if this is a reasonable difficulty

Reply:
Seems.good to me.